### ISSUE TITLE
False positive: webpack loaders in "modules"

### ISSUE THREAD

**Issue by clar-cmp** [depcheck/depcheck#572]
I'm not sure if the current code checks for an older version of webpack, but we've been using webpack loaders like so:

```js
    module: {
        rules: [
            {
                exclude: /\/node_modules\//,
                test: /\.[jt]s$/,
                use: [
                    {
                        loader: 'cache-loader',
                    },
                    {
                        loader: 'babel-loader',
                        options: {
                            cacheDirectory: true,
                        },
                    },
                ],
            },
        ],
    },
```

And these aren't being counted as used.

**Comment by clar-cmp** [comment]
Still an issue

**Comment by rumpl** [comment]
PRs are welcome 🤗
